src/hb-algs.hh

```diff
@@ -645,16 +645,26 @@ hb_unsigned_mul_overflows (unsigned int count, unsigned int size)
   return (size > 0) && (count >= ((unsigned int) -1) / size);
 }
 
+static inline bool
+hb_signed_mul_overflows (int x, int y, int &result)
```

Comment:
I would use `signed` instead of `int` here but that's just my new taste.

Reply:
It is understandable, done.